### ISSUE TITLE
Candles remove light when lit

### ISSUE THREAD

**Issue by WallyCat** [gri573/rethinking-voxels#80]
Instead of adding to light when candles are lit, they take away light, and vice-versa.

https://user-images.githubusercontent.com/104689705/220209623-f71852b1-0cea-42ad-ba15-4a0c389f3ef7.mp4

**Comment by gri573** [comment]
that's just setting the light colour to its own colour at its location, it's fundamental to how lights work in basic light propagation mode

**Comment by gri573** [comment]
actually, I think I can improve it

**Comment by WallyCat** [comment]
yeah, when i was testing with all different colors of candles before i posted this, i realized that it's just outputting the color of the candle, but no matter what color it always looked to take away light from the area rather than add onto. i appreciate you looking into how you could improve it, 'cuz i really do adore the work you've done here.

**Comment by gri573** [comment]
fixed by changing how basic light propagation works altogether. maybe it is more accurate to say that the issue was removed along with basic light propagation, which was replaced by per-block RT lighting
